docs/source/slots/anatomy.md

```diff
@@ -0,0 +1,58 @@
+# Slots anatomy
+
+The slots are insertion points in the Volto rendering tree structure. You can add a
+component, along with its configuration, if any and it will be rendered in that
+insertion point. You can control in which route do you want that element appear as well
+as the order (position) of the items that you add in the in the slots. Slots are named,
+so you can add in the configuration object:
+
+```js
+export const slots = {
+  aboveContentTitle: [
+    // List of components (might have config too, maybe in `data` property)
+    { path: '/', component: 'Component', data: {} },
+    // It can include blocks too (makes sense when we will be able to save them)
+    { path: '/', '@type': 'text' },
+    { path: '/', '@type': 'image' },
+  ],
+};
+```
+
+## Slots
+
+- aboveContentTitle
+- belowContentTitle
+- aboveContentBody
+- belowContentBody
+- footer
+
+- leftAsideSlot
+- rightAsideSlot
+
+- afterApp
+- afterToolbar
+
+- htmlHead
+- htmlBeforeBody
+- htmlAfterBody
+
```

Comment:
I initially identified these slots (and their insertion positions), did I miss something? We can add more later, for sure. @nzambello the ones referring to your "Macros" PR are there as well.

Reply:
I think Sidebar is a good candidate, as well.

Though I can definitely see Martjin's point for the toolbar: if extra content is needed there, then maybe we need a system of definable actions. For the Toolbar, my wishlist include new buttons in the main toolbar, but also now components available for the "More" menu.